modules/client/variables.tf

```
@@ -66,6 +66,12 @@ variable "server_certificate_arn" {
   type        = string
 }
 
+variable "split_tunnel" {
+  description = "Enables/disables split tunnel on the Client VPN."
+  type        = string
+  default     = "false"
```

TF 0.12 lets us get a little more restricitve, and added several additional variable types.  Instead of string, we should probably make this a bool

```suggestion
  type        = bool
  default     = false
```